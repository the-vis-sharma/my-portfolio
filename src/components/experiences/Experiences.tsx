import { forwardRef } from "react";
import { AccessTimeFilled } from "@mui/icons-material";
import {
  Timeline,
  TimelineConnector,
  TimelineContent,
  TimelineDot,
  TimelineItem,
  TimelineOppositeContent,
  TimelineSeparator,
} from "@mui/lab";
import { Box, Theme, Typography, useMediaQuery } from "@mui/material";
import CompanyDetail from "./CompanyDetail";
import CompanyList from "./data";

const Experiences = forwardRef((_, ref) => {
  const isTabletOrSmaller = useMediaQuery((theme: Theme) =>
    theme.breakpoints.down("md")
  );
  return (
    <Box
      ref={ref}
      id="experiences-section"
      sx={{
        p: 4,
        pt: 8,
      }}
    >
      <Typography
        variant="h3"
        textAlign="center"
        gutterBottom
        color="primary"
        fontWeight="500"
      >
        Experiences
      </Typography>

      {!isTabletOrSmaller ? (
        <Timeline>
          {CompanyList.map((company) => (
            <TimelineItem key={company.name}>
              <TimelineOppositeContent
                sx={{ m: "auto 0", flex: 0 }}
                variant="body2"
                color="text.secondary"
              >
                <Box sx={{ width: "100px" }}>{company.duration}</Box>
              </TimelineOppositeContent>
              <TimelineSeparator>
                <TimelineConnector />
                <TimelineDot color="primary">
                  <AccessTimeFilled />
                </TimelineDot>
                <TimelineConnector />
              </TimelineSeparator>
              <TimelineContent sx={{ py: "12px", px: 2 }}>
                <CompanyDetail {...company} />
              </TimelineContent>
            </TimelineItem>
          ))}
        </Timeline>
      ) : (
        CompanyList.map((company) => (
          <CompanyDetail key={company.name} {...company} />
        ))
      )}
    </Box>
  );
});

export default Experiences;
