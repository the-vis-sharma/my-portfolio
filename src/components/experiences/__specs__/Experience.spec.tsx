import { cleanup, render } from "@testing-library/react";
import Experiences from "../Experiences";

describe("test experiences component", () => {
  const mockUseMediaQuery = (
    width: number,
    predicate: (q: string) => boolean
  ) => {
    Object.defineProperty(window, "innerWidth", {
      writable: true,
      configurable: true,
      value: width,
    });
    window.matchMedia = jest.fn().mockImplementation((query) => ({
      matches: predicate(query),
      media: query,
      onchange: null,
      addListener: jest.fn(),
      removeListener: jest.fn(),
    }));
  };

  afterEach(() => {
    cleanup();
    jest.clearAllMocks();
  });

  it("verify snapshot for larger display", () => {
    mockUseMediaQuery(1600, (query) => !!query);
    const { asFragment } = render(<Experiences />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("verify snapshot for tablet or lesser size display", () => {
    mockUseMediaQuery(400, (query) => !!query);
    const { asFragment } = render(<Experiences />);
    expect(asFragment()).toMatchSnapshot();
  });
});
