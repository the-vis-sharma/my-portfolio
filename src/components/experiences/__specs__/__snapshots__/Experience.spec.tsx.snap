// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test experiences component verify snapshot for larger display 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-eeo3ga"
    id="experiences-section"
  >
    <h3
      class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1oe6gg4-MuiTypography-root"
    >
      Experiences
    </h3>
    <div
      class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-1wqstqh-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiBox-root css-out6mg"
        >
          <picture>
            <source
              srcset="test-file-stub"
              type="image/webp"
            />
            <source
              srcset="test-file-stub"
              type="image/jpeg"
            />
            <img
              alt="Amazon Logo"
              src="test-file-stub"
              style="height: 40px;"
            />
          </picture>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1wwkujw-MuiTypography-root"
            >
              Software Dev Engineer
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-37mko6-MuiTypography-root"
            >
              Bangalore, IN
            </h6>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
        >
          Amazon.com, Inc. is an American multinational technology company which focuses on e-commerce, cloud computing, digital streaming, and artificial intelligence. It is one of the Big Five companies in the U.S. information technology industry, along with Google, Apple, Meta, and Microsoft.
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
        >
          ACCOMPLISHMENTS
        </h6>
        <ul>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Working at Amazon Device OS Team for building new frontend framework for Amazon devices by extending React Native to support Amazon Devices operating system like Amazon Alexa Echo, Echo Show, FireTV etc.
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Building core and complex common React Native components library for 2P and 3P developers for Amazon Devices operating system.
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Build a SVG rendering library from Scratch like react-native-svg for Amazon Devices operating system using React Native and C++.
            </p>
          </li>
        </ul>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root Mui-disabled  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
          disabled=""
          tabindex="-1"
          type="button"
        >
          Jan 2022 - Present
        </button>
      </div>
    </div>
    <div
      class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-1wqstqh-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiBox-root css-out6mg"
        >
          <picture>
            <source
              srcset=""
              type="image/webp"
            />
            <source
              srcset="test-file-stub"
              type="image/jpeg"
            />
            <img
              alt="Thoughtworks Logo"
              src="test-file-stub"
              style="height: 40px;"
            />
          </picture>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1wwkujw-MuiTypography-root"
            >
              Full Stack Developer
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-37mko6-MuiTypography-root"
            >
              Bangalore, IN
            </h6>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
        >
          Thoughtworks is a leading global technology consultancy that integrates strategy, design and software engineering to enable enterprises and technology disruptors across the globe to thrive as modern digital businesses.
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
        >
          ACCOMPLISHMENTS
        </h6>
        <ul>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Building a e-commerce website from scratch using ReactJS, Spring Boot, Microservices, Material Design etc with a team of 15 members
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Done initial setup of UI repo as Mono repo using yarn workspace with eslint, stylelint, prettier, husky pre-commit hook, lint-staged and jest for testing
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Created Backend API for product details page with basic product details, quanitity, prices, dealers etc
            </p>
          </li>
        </ul>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root Mui-disabled  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
          disabled=""
          tabindex="-1"
          type="button"
        >
          Oct 2021 - Jan 2022
        </button>
      </div>
    </div>
    <div
      class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-1wqstqh-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiBox-root css-out6mg"
        >
          <picture>
            <source
              srcset="test-file-stub"
              type="image/webp"
            />
            <source
              srcset="test-file-stub"
              type="image/jpeg"
            />
            <img
              alt="IBM Logo"
              src="test-file-stub"
              style="height: 40px;"
            />
          </picture>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1wwkujw-MuiTypography-root"
            >
              Full Stack Developer
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-37mko6-MuiTypography-root"
            >
              Bangalore, IN
            </h6>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
        >
          International Business Machines Corporation is an American multinational technology company headquartered in Armonk, New York, with operations in over 170 countries.
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
        >
          ACCOMPLISHMENTS
        </h6>
        <ul>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Working as a full stack developer on a large-scale, cloud-native, and distributed crew logistics solution with tools & technologies like Spring Boot, React.Js, Groovy, Micro-services, Redis, Cassandra, Kafka, Kibana, Grafana, Portainer in an agile team.
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Improved live tracking speed by 15% by optimising code
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Improved scalability, fault tolerance, and reliability by designing micro-service architecture after event storming with stakeholders.
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Completed 400+ user stories and implemented 15+ new features.
            </p>
          </li>
        </ul>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root Mui-disabled  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
          disabled=""
          tabindex="-1"
          type="button"
        >
          Jun 2019 - Oct 2021
        </button>
      </div>
    </div>
    <div
      class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-1wqstqh-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiBox-root css-out6mg"
        >
          <picture>
            <source
              srcset="test-file-stub"
              type="image/webp"
            />
            <source
              srcset="test-file-stub"
              type="image/jpeg"
            />
            <img
              alt="C-DAC Logo"
              src="test-file-stub"
              style="height: 40px;"
            />
          </picture>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1wwkujw-MuiTypography-root"
            >
              Machine Learning Trainee
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-37mko6-MuiTypography-root"
            >
              Jaipur, IN
            </h6>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
        >
          The Centre for Development of Advanced Computing is an Indian autonomous scientific society, operating under the Ministry of Electronics and Information Technology.
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
        >
          ACCOMPLISHMENTS
        </h6>
        <ul>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Gained experience in machine learning using python by working on a project of weather prediction android app which shows weather forecast
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Used multi-dimension linear regression model for prediction using 100 years of historical data set using information retrieval
            </p>
          </li>
        </ul>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root Mui-disabled  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
          disabled=""
          tabindex="-1"
          type="button"
        >
          May 2018 - Jul 2018
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`test experiences component verify snapshot for tablet or lesser size display 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-eeo3ga"
    id="experiences-section"
  >
    <h3
      class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1oe6gg4-MuiTypography-root"
    >
      Experiences
    </h3>
    <div
      class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-1wqstqh-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiBox-root css-out6mg"
        >
          <picture>
            <source
              srcset="test-file-stub"
              type="image/webp"
            />
            <source
              srcset="test-file-stub"
              type="image/jpeg"
            />
            <img
              alt="Amazon Logo"
              src="test-file-stub"
              style="height: 40px;"
            />
          </picture>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1wwkujw-MuiTypography-root"
            >
              Software Dev Engineer
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-37mko6-MuiTypography-root"
            >
              Bangalore, IN
            </h6>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
        >
          Amazon.com, Inc. is an American multinational technology company which focuses on e-commerce, cloud computing, digital streaming, and artificial intelligence. It is one of the Big Five companies in the U.S. information technology industry, along with Google, Apple, Meta, and Microsoft.
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
        >
          ACCOMPLISHMENTS
        </h6>
        <ul>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Working on Next Big Thing
            </p>
          </li>
        </ul>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root Mui-disabled  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
          disabled=""
          tabindex="-1"
          type="button"
        >
          Jan 2022 - Present
        </button>
      </div>
    </div>
    <div
      class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-1wqstqh-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiBox-root css-out6mg"
        >
          <picture>
            <source
              srcset=""
              type="image/webp"
            />
            <source
              srcset="test-file-stub"
              type="image/jpeg"
            />
            <img
              alt="Thoughtworks Logo"
              src="test-file-stub"
              style="height: 40px;"
            />
          </picture>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1wwkujw-MuiTypography-root"
            >
              Full Stack Developer
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-37mko6-MuiTypography-root"
            >
              Bangalore, IN
            </h6>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
        >
          Thoughtworks is a leading global technology consultancy that integrates strategy, design and software engineering to enable enterprises and technology disruptors across the globe to thrive as modern digital businesses.
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
        >
          ACCOMPLISHMENTS
        </h6>
        <ul>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Building a e-commerce website from scratch using ReactJS, Spring Boot, Microservices, Material Design etc with a team of 15 members
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Done initial setup of UI repo as Mono repo using yarn workspace with eslint, stylelint, prettier, husky pre-commit hook, lint-staged and jest for testing
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Created Backend API for product details page with basic product details, quanitity, prices, dealers etc
            </p>
          </li>
        </ul>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root Mui-disabled  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
          disabled=""
          tabindex="-1"
          type="button"
        >
          Oct 2021 - Jan 2022
        </button>
      </div>
    </div>
    <div
      class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-1wqstqh-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiBox-root css-out6mg"
        >
          <picture>
            <source
              srcset="test-file-stub"
              type="image/webp"
            />
            <source
              srcset="test-file-stub"
              type="image/jpeg"
            />
            <img
              alt="IBM Logo"
              src="test-file-stub"
              style="height: 40px;"
            />
          </picture>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1wwkujw-MuiTypography-root"
            >
              Full Stack Developer
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-37mko6-MuiTypography-root"
            >
              Bangalore, IN
            </h6>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
        >
          International Business Machines Corporation is an American multinational technology company headquartered in Armonk, New York, with operations in over 170 countries.
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
        >
          ACCOMPLISHMENTS
        </h6>
        <ul>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Working as a full stack developer on a large-scale, cloud-native, and distributed crew logistics solution with tools & technologies like Spring Boot, React.Js, Groovy, Micro-services, Redis, Cassandra, Kafka, Kibana, Grafana, Portainer in an agile team.
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Improved live tracking speed by 15% by optimising code
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Improved scalability, fault tolerance, and reliability by designing micro-service architecture after event storming with stakeholders.
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Completed 400+ user stories and implemented 15+ new features.
            </p>
          </li>
        </ul>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root Mui-disabled  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
          disabled=""
          tabindex="-1"
          type="button"
        >
          Jun 2019 - Oct 2021
        </button>
      </div>
    </div>
    <div
      class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-1wqstqh-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiBox-root css-out6mg"
        >
          <picture>
            <source
              srcset="test-file-stub"
              type="image/webp"
            />
            <source
              srcset="test-file-stub"
              type="image/jpeg"
            />
            <img
              alt="C-DAC Logo"
              src="test-file-stub"
              style="height: 40px;"
            />
          </picture>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1wwkujw-MuiTypography-root"
            >
              Machine Learning Trainee
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-37mko6-MuiTypography-root"
            >
              Jaipur, IN
            </h6>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-1oetjkv-MuiTypography-root"
        >
          The Centre for Development of Advanced Computing is an Indian autonomous scientific society, operating under the Ministry of Electronics and Information Technology.
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
        >
          ACCOMPLISHMENTS
        </h6>
        <ul>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Gained experience in machine learning using python by working on a project of weather prediction android app which shows weather forecast
            </p>
          </li>
          <li
            style="text-align: justify;"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-1ip9kmw-MuiTypography-root"
            >
              Used multi-dimension linear regression model for prediction using 100 years of historical data set using information retrieval
            </p>
          </li>
        </ul>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
      />
      <div
        class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root Mui-disabled  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
          disabled=""
          tabindex="-1"
          type="button"
        >
          May 2018 - Jul 2018
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
